test/mode/modeVisualLine.test.ts

```diff
@@ -316,6 +316,13 @@ suite('Mode Visual Line', () => {
     });
   });
 
+  newTest({
+    title: 'Vp updates register content',
+    start: ['|hello', 'world'],
+    keysPressed: 'ddVpP',
+    end: ['|world', 'hello', ''],
```

Comment:
can you add a comment here mentioning that this test is actually not truly "correct"

Reply:
done :)